### PR TITLE
feat: #152 landing page improvements

### PR DESCRIPTION
**Pull request by buenaflor**
Landing page improvements based on #193 and #152 including Margins, Tailblocks. NavBar is not yet done.

Colors have not been considered in this PR. 

It's best probably to see the site first before reviewing the code

Tips for review:
Most of the components are directly from tailblocks.  I would focus on changes in:
 - `LandingPage.tsx`
 - `Features.tsx`
 - `ButtonLink.tsx`
 - `globals.css`

EDIT:

Not included:
 - Logos -> issues with styling them directly in code, for testing purposes these worked for me: https://fonts.google.com/icons
 - Map -> issues with overlapping navbar, needs more investigation (zIndex didn't help here)

All in all I'd like to leave it as it is now since there are other issues that have higher prio

## Basics

- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.
- [ ] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is conforming to [our Documentation Guidelines](/doc/documentation.md)
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to our Coding Guidelines
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

### CODE REVIEW

**Comment by markus2330** [comment]
jenkins build please

**Comment by markus2330** [comment]
Both locally and on pr.permaplant.net I only see a blank page. Does this require a more recent npm? Can this be avoided?

**Comment by buenaflor** [comment]
Did you try it on incognito mode? Might be a cache thing because I added a new vite plugin for loading svgs directly

**Comment by markus2330** [comment]
It works now! :heart:

**Comment by buenaflor** [comment]
Updated comment on what's missing - it is ready for review

File: frontend/src/components/Layout/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+const Navbar = () => {
+  return (
+    <nav className="fixed top-0 left-0 z-50 w-full border-b-[0.5px] border-gray-200 bg-white dark:border-gray-700 dark:bg-neutral-100-dark">
+      <div className="mx-auto flex max-w-screen-xl flex-wrap items-center justify-between p-4">
+        <div className="flex space-x-16">
+          <div className="flex space-x-6">
+            <a href="/" className="flex items-center">
+              <span className="self-center whitespace-nowrap text-2xl font-semibold dark:text-white">
+                PermaPlanT
```
Comment (markus2330) [inline frontend/src/components/Layout/Navbar.tsx:12]:
```suggestion
                PermaplanT
```

File: frontend/src/components/Layout/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+              </li>
+              <li>
+                <ButtonLink title="Seeds" to="/seeds" />
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div className="flex md:order-2">
+          <div className="flex items-center space-x-4">
+            <a href="https://github.com/ElektraInitiative/PermaPlant">GitHub</a>
```
Comment (markus2330) [inline frontend/src/components/Layout/Navbar.tsx:32]:
```suggestion
            <a href="https://github.com/ElektraInitiative/PermaplanT">GitHub</a>
```

**Comment by markus2330** [comment]
jenkins build please

File: frontend/src/components/Button/ButtonLink.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { Link } from 'react-router-dom';
```
Comment (markus2330) [inline frontend/src/components/Button/ButtonLink.tsx:1]: Please add a bit documentation for each component which might be reused (CTA etc. is okay to not have much documentation).

**Review by markus2330 — APPROVED** [review]
Please add a bit documentation to the components.

@absurd-turtle can you then readd the map?

**Review by badnames — APPROVED** [review]
Quite a lot of changes, so I could not verify everything.
But everything I've seen thus far looks pretty good to me.

Just one little thing: maybe add some documentation to all newly added components describing their purpose, like you did in NavContainer for example.

**Comment by markus2330** [comment]
Great work! Let us add more documentation when the proper tools are introduced.

**Pull request merged**